.idea/misc.xml

```diff
@@ -7,6 +7,7 @@
     <option name="originalFiles">
       <list>
         <option value="$PROJECT_DIR$/optaweb-tsp-planner/pom.xml" />
+        <option value="$PROJECT_DIR$/optaweb-tsp-ui/pom.xml" />
```

Comment:
@yurloc .idea files shouldn't be checked in. Please use the same .gitignore file as the optaplanner repository.

Please use an *.adoc for the readme, like the he optaplanner repository.
Please use the same license file with the same name, like in the optaplanner repository's root dir.

Reply:
I used the default flavor of [.gitignore for IDEA](https://www.gitignore.io/api/intellij). This flavor only ignores files under`.idea/` dir that are never intended to be version-controlled. There are other flavors that ignore [.iml files](https://www.gitignore.io/api/intellij+iml) too or [everything under .idea/ dir](https://www.gitignore.io/api/intellij+all) as well.

I thought it might be convenient for other developers that use IDEA to get the project up and running if the basic IDEA configuration is included in the repository. Some people are convinced that the repository [should be kept clean](https://github.com/joeblau/gitignore.io/issues/186#issuecomment-249319722) of **any IDE**-specific metadata, which I think is a legitimate approach. If that's your opinion too and unless you change your mind, I'm going to remove and ignore the whole `.idea/` directory.